lamindb_setup/_load_instance.py

```diff
@@ -14,8 +14,10 @@
 from .dev._settings_store import instance_settings_file
 from .dev._hub_client import call_with_fallback
 from .dev._hub_crud import sb_select_account_handle_name_by_lnid
+from .dev.cloud_sqlite_locker import InstanceLockedException, unlock_on_exception
 
 
+@unlock_on_exception
```

Comment:
Could you rename to `unlock_cloud_sqlite_upon_exception`?

Reply:
Done.